Google.Api.Generator/Generation/UnitTestCodeGeneration.cs

```diff
@@ -298,17 +298,18 @@ public class Signature
                 private string SyncMethodName => Method.SyncMethodName + (_index is int index ? (index + 1).ToString() : "");
                 private string AsyncMethodName => $"{Method.AsyncMethodName.Substring(0, Method.AsyncMethodName.Length - 5)}{(_index is int index ? (index + 1).ToString() : "")}Async";
 
-                private ExpressionSyntax Access(LocalDeclarationStatementSyntax request, string name, bool isDeprecated)
+                private ExpressionSyntax Access(MemberAccessExpressionSyntax access, bool isDeprecated)
                 {
-                    var access = request.Access(name);
                     if (isDeprecated)
                     {
                         access = access.WithName(access.Name.WithIdentifier(access.Name.Identifier.WithPragmaWarning(PragmaWarnings.Obsolete)));
                     }
                     return access;
                 }
 
-                private IEnumerable<object> SigArgs => _sig.Fields.Select(field => Access(_def.Request, field.PropertyName, field.IsDeprecated));
+                private IEnumerable<object> SigArgs => _sig.Fields.Select(field => Access(
+                    field.Descs.Skip(1).Aggregate(_def.Request.Access(field.Descs.First().CSharpPropertyName()), (acc, desc) => (MemberAccessExpressionSyntax)acc.Access(desc.CSharpPropertyName())),
```

Comment:
That's a really dense line of code to go uncommented...

Any chance it could be laid out over more lines, as well as with some explanation?

Reply:
(After looking at it for a while in the context of knowing the change, I think I understand... but it took a while...)

Reply:
One comment coming up...